### PR TITLE
Added support for SVG images

### PR DESCRIPTION
**Pull request by niftylettuce**

### CODE REVIEW

**Comment by niftylettuce** [comment]
Might need to add tests for it.

File: src/replaceBase64Images.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
  var replaceBase64Images = function(html, getCid) {
-   return html.replace(/(<img[\s\S]*? src=")data:(image\/(?:png|jpe?g|gif));base64,([\s\S]*?)("[\s\S]*?>)/g, function(g, start, mimeType, base64, end) {
+   return html.replace(/(<img[\s\S]*? src=")data:(image\/(?:png|jpe?g|gif|svg\+xml));base64,([\s\S]*?)("[\s\S]*?>)/g, function(g, start, mimeType, base64, end) {
```
Comment (bradvogel) [inline src/replaceBase64Images.js]: Please add unit tests.

File: src/plugin.js [hunk 2]
```diff
@@ -12,9 +12,10 @@ var plugin = function(mail, done) {
     var attachments = {};
 
     html = replaceBase64Images(html, function(mimeType, base64) {
+
       if (attachments[base64]) return attachments[base64].cid;
 
-      var randomCid = crypto.randomBytes(8).toString('hex');
+      var randomCid = crypto.randomBytes(8).toString('hex') + '@nodemailer.com';
```
Comment (bradvogel) [inline src/plugin.js]: It's interesting that Gmail's own Content-IDs don't have an @: `Content-ID: <ii_iu2vlgi02_157aa636a6891103>`. Just curious, what mail client didn't show the image for you when you tested Content-IDs without @?

**Comment by bradvogel** [comment]
Closing due to inactivity.

**Pull request closed without merging**